### ISSUE TITLE
[FEATURE] Allow changing app launch transitions

### ISSUE THREAD

**Issue by SeanDylan** [Deletescape-Media/Lawnchair#3961]
### Describe the feature

This feature was never really used but i would like to see some opening animations back and creating new emulated animations from android pie to upside down cake some may notice it if you slow it down but others won't

### Why would it be useful to add?

More smoother animations and feel of pixel launcher

### Did the feature exist in Lawnchair v2? (Play Store version)

Yes

### Example(s)

![Screenshot_20240112-172243](https://github.com/LawnchairLauncher/lawnchair/assets/104182251/ca30043b-62d6-4c3c-85c0-3adc41dae4ad)

### Additional context

Since stock android animations cannot be implemented by default we can just emulate the animations since it's not possible to actually implement this as non emulated (yes this is possible with quickswitch but i'm talking about non-rooted)

**Comment by Alpointernet** [comment]
Yeah that would be great. After OneUI 6, the app opening animation became "reveal" for me. So I would like to change that.
